src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1212,6 +1210,14 @@ uct_ib_mlx5_devx_device_mem_free(uct_md_h uct_md, uct_mem_h memh)
         status = UCS_ERR_BUSY;
     }
 
+    uct_ib_mlx5_devx_obj_destroy(ib_memh->dm_addr_dvmr, "DM Address, KSM");
+
+    ret = munmap(ib_memh->address,
+                 ib_memh->mrs[UCT_IB_MR_DEFAULT].super.ib->length);
+    if (ret != 0) {
+        ucs_warn("munmap() failed: %m");
```

Comment:
print address and length
can use helper variable (or inline function, if can reduce code dup) for `ib_memh->mrs[UCT_IB_MR_DEFAULT].super.ib->length`